gas/testsuite/ChangeLog.COREV

```diff
@@ -1,3 +1,312 @@
+	* gas/riscv/cv-postinc-lb.d: Likewise.
+	* gas/riscv/cv-postinc-lbpost.d: Likewise.
+	* gas/riscv/cv-postinc-lbpost.s: Likewise.
+	* gas/riscv/cv-postinc-lbrr.d: Likewise.
+	* gas/riscv/cv-postinc-lbrrpost.d: Likewise.
+	* gas/riscv/cv-postinc-lbrrpost.s: Likewise.
+	* gas/riscv/cv-postinc-lbrr.s: Likewise.
+	* gas/riscv/cv-postinc-lb.s: Likewise.
+	* gas/riscv/cv-postinc-lbu.d: Likewise.
+	* gas/riscv/cv-postinc-lbupost.d: Likewise.
+	* gas/riscv/cv-postinc-lbupost.s: Likewise.
+	* gas/riscv/cv-postinc-lburr.d: Likewise.
+	* gas/riscv/cv-postinc-lburrpost.d: Likewise.
+	* gas/riscv/cv-postinc-lburrpost.s: Likewise.
+	* gas/riscv/cv-postinc-lburr.s: Likewise.
+	* gas/riscv/cv-postinc-lbu.s: Likewise.
+	* gas/riscv/cv-postinc-lh.d: Likewise.
+	* gas/riscv/cv-postinc-lhpost.d: Likewise.
+	* gas/riscv/cv-postinc-lhpost.s: Likewise.
+	* gas/riscv/cv-postinc-lhrr.d: Likewise.
+	* gas/riscv/cv-postinc-lhrrpost.d: Likewise.
+	* gas/riscv/cv-postinc-lhrrpost.s: Likewise.
+	* gas/riscv/cv-postinc-lhrr.s: Likewise.
+	* gas/riscv/cv-postinc-lh.s: Likewise.
+	* gas/riscv/cv-postinc-lhu.d: Likewise.
+	* gas/riscv/cv-postinc-lhupost.d: Likewise.
+	* gas/riscv/cv-postinc-lhupost.s: Likewise.
+	* gas/riscv/cv-postinc-lhurr.d: Likewise.
+	* gas/riscv/cv-postinc-lhurrpost.d: Likewise.
+	* gas/riscv/cv-postinc-lhurrpost.s: Likewise.
+	* gas/riscv/cv-postinc-lhurr.s: Likewise.
+	* gas/riscv/cv-postinc-lhu.s: Likewise.
+	* gas/riscv/cv-postinc-lw.d: Likewise.
+	* gas/riscv/cv-postinc-lwpost.d: Likewise.
+	* gas/riscv/cv-postinc-lwpost.s: Likewise.
+	* gas/riscv/cv-postinc-lwrr.d: Likewise.
+	* gas/riscv/cv-postinc-lwrrpost.d: Likewise.
+	* gas/riscv/cv-postinc-lwrrpost.s: Likewise.
+	* gas/riscv/cv-postinc-lwrr.s: Likewise.
+	* gas/riscv/cv-postinc-lw.s: Likewise.
+	* gas/riscv/cv-postinc-march-rv32i-xcorev.d: Likewise.
+	* gas/riscv/cv-postinc-march-rv32i-xcorev.s: Likewise.
+	* gas/riscv/cv-postinc-sb.d: Likewise.
+	* gas/riscv/cv-postinc-sbpost.d: Likewise.
+	* gas/riscv/cv-postinc-sbpost.s: Likewise.
+	* gas/riscv/cv-postinc-sbrr.d: Likewise.
+	* gas/riscv/cv-postinc-sbrrpost.d: Likewise.
+	* gas/riscv/cv-postinc-sbrrpost.s: Likewise.
+	* gas/riscv/cv-postinc-sbrr.s: Likewise.
+	* gas/riscv/cv-postinc-sb.s: Likewise.
+	* gas/riscv/cv-postinc-sh.d: Likewise.
+	* gas/riscv/cv-postinc-shpost.d: Likewise.
+	* gas/riscv/cv-postinc-shpost.s: Likewise.
+	* gas/riscv/cv-postinc-shrr.d: Likewise.
+	* gas/riscv/cv-postinc-shrrpost.d: Likewise.
+	* gas/riscv/cv-postinc-shrrpost.s: Likewise.
+	* gas/riscv/cv-postinc-shrr.s: Likewise.
+	* gas/riscv/cv-postinc-sh.s: Likewise.
+	* gas/riscv/cv-postinc-sw.d: Likewise.
+	* gas/riscv/cv-postinc-swpost.d: Likewise.
+	* gas/riscv/cv-postinc-swpost.s: Likewise.
+	* gas/riscv/cv-postinc-swrr.d: Likewise.
+	* gas/riscv/cv-postinc-swrrpost.d: Likewise.
+	* gas/riscv/cv-postinc-swrrpost.s: Likewise.
+	* gas/riscv/cv-postinc-swrr.s: Likewise.
+	* gas/riscv/cv-postinc-sw.s: Likewise.
+
```

Comment:
Please see previous comment about ChangeLogs.

Really great that you have a lot of tests, however, I have a couple of comments on the testing:
1. I don't think it is necessary to test failure modes with every operand with each opcode. As seen in opcodes/riscv-opc.c a lot of the instructions use the same operand types. A good approach would be to test a representative for each failure mode. That way you don't get combinatorial explosion in the number of tests.
2. If point (1.) is implemented there is no need for the instruction name in the failure tests as this does not give information the intention of the test. For consistency I will add fail to existing CORE-V failure tests. If you agree we can stick to something like 'cv-postinc-fail-01.s', 'cv-alu-fail-01.s' etc. as our naming convention?

Reply:
Please see #19 for example of these changes implemented for ALU ops.